yarn-project/cli/package.json

```diff
@@ -35,6 +35,7 @@
   },
   "dependencies": {
     "@aztec/aztec.js": "workspace:^",
+    "@aztec/circuits.js": "workspace:^",
```

Comment:
What do we need `circuits.js` here for? Is this for something that should be exported through `aztec.js`

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/2600/commits/00f0060bd4d0df46408534c1da56f292cd16e90a